libraries/stumbler/src/main/java/org/mozilla/mozstumbler/service/stumblerthread/motiondetection/MotionSensor.java

```diff
@@ -65,6 +65,7 @@ private void setTypeFromPrefs() {
     // This class can use this event to update its state
     public void scannerFullyStopped() {
         setTypeFromPrefs();
+        mFalsePositiveFilter.reset();
```

Comment:
I'd like a test that exercises the motion detector state to clearly see what scenarios we can expect `reset()` vs `setTime()` to be invoked.